### ISSUE TITLE
A11y: The Combobox Input outline is not visible when it receives focus.

### ISSUE THREAD

**Issue by hkshetry** [SAP/fundamental-styles#5951]
**Describe the bug**
A11y: The Combobox Input outline is not visible when it receives focus.

> Is this issue related to a specific component?
yes it is Combobox Input  and date picker

> What version of the Fundamental Library Styles are you using?
 v0.38.0
> What offering/product do you work on? Any pressing ship or release dates we should be aware of?
 v0.38.0
> What front-end framework are you implementing in, e.g., Angular, React, etc.?
Just fundamental style

**To Reproduce**
1. Go to https://sap.github.io/fundamental-styles/?path=/docs/sap-fiori-patterns-combobox-input--docs
2. Click on type some work
3.  use keyword navigation
4. See focus outline is missing when the focus is received

**Expected behavior**

**Screenshots**

![Image](https://github.com/user-attachments/assets/3a464e73-8977-4801-b1f6-773f4dc24678)

**Comment by droshev** [comment]
@InnaAtanasova Can you check this issue?
